<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minesweeper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>185</width>
      <height>185</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_mines">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_look">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_time">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>186</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionPrimary"/>
    <addaction name="actionMid"/>
    <addaction name="actionSenior"/>
    <addaction name="actionCustom"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionPrimary">
   <property name="text">
    <string>Beginner</string>
   </property>
  </action>
  <action name="actionMid">
   <property name="text">
    <string>Intermediate</string>
   </property>
  </action>
  <action name="actionSenior">
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Customize</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
